llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -4270,6 +4271,25 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
     setOrigin(&I, PtrSrcOrigin);
   }
 
+  void checkPermilvarIndexShadow(IRBuilder<> &IRB, Value *Idx, Instruction *I) {
```

Comment:
Assuming this will be used by handleAVXVpermi2var (current patch) and handleAVXVpermilvar (future patch?), please rename so that it's not Permilvar-specific, and also to include AVX.

Reply:
done